src/interfaces/chain.h

```diff
@@ -189,8 +180,10 @@ class Chain
     //! Check if transaction has descendants in mempool.
     virtual bool hasDescendantsInMempool(const uint256& txid) = 0;
 
-    //! Relay transaction.
-    virtual void relayTransaction(const uint256& txid) = 0;
+    //! Transaction is added to memory pool, if the transdaction fee is below the
+    //! amount specified by absurdfee, and broadcast to all nodes.
+    //! Return false if the transaction could not be added due to the fee or for another reason.
+    virtual bool broadcastTransaction(const CTransactionRef& tx, uint256& txid, std::string& err_string, const CAmount& absurdfee, bool wait_callback) = 0;
```

Comment:
nit: I'd prefer to standardise on the name `max_tx_fee` (rather than `max_tx_fee`, `highfee`, `absurdfee`, etc)